service/src/main/java/org/jboss/sbomer/service/feature/sbom/errata/event/release/ReleaseStandardAdvisoryEventsListener.java

```diff
@@ -629,11 +626,13 @@ protected Sbom saveReleaseManifestForDockerGeneration(
                                         .getComponent()
                                         .getDescription()
                                         .contains(buildManifest.getRootPurl())) {
-                            // FIXME: Result of String.replace() is ignored
                             manifestBom.getMetadata()
                                     .getComponent()
-                                    .getDescription()
-                                    .replace(buildManifest.getRootPurl(), rebuiltPurl);
+                                    .setDescription(
+                                            manifestBom.getMetadata()
+                                                    .getComponent()
+                                                    .getDescription()
+                                                    .replace(buildManifest.getRootPurl(), rebuiltPurl));
```

Comment:
@vibe13 I think this should fix the issue where description wasn't actually being changed.

Reply:
I have fixed it as part of https://github.com/project-ncl/sbomer/pull/1164